app/scripts/directives/readFile.js

```diff
@@ -0,0 +1,39 @@
+angular
+  .module('confRegistrationWebApp')
+  .directive('readFile', function($timeout, $q) {
```

Comment:
Can you change the functions in this file to arrow functions?

Reply:
done